packages/libretro/reicast/package.mk

```diff
@@ -34,6 +34,15 @@ PKG_LONGDESC="Reicast is a multiplatform Sega Dreamcast emulator"
 PKG_IS_ADDON="no"
 PKG_AUTORECONF="no"
 
+pre_configure_target() {
```

Comment:
you can replace these lines with `strip_lto`

Reply:
Done. Good catch, thanks!